Ternary_Search/Ternary_Search.php

```diff
@@ -0,0 +1,55 @@
+ <?php
+
+    /* Ternary search is a searching technique much like binary search but instead of dividing the given array in two parts,     it divides it in 3 parts. It order to search, it needs the array to be sorted.*/
```

Comment:
Please split the comment into multiple lines. One line can't contain more than 80 characters.